src/Common/MainController.h

```diff
@@ -315,7 +315,7 @@ public slots:
 
     QScopedPointer<Geo::IpToLocationResolver> ipToLocationResolver;
 
-    QList<Recorder::JamRecorder> jamRecorders;
+    QList<Recorder::JamRecorder *> jamRecorders;
```

Comment:
Hi,

OK, after trying all evening with the non-pointer list, I gave up and spent a couple of minutes adding deletes to the destructors.  And some verbose logging...  It looks like it cleans up okay.

Let me know if it looks okay :).

-- Peter